packages/legend-query-builder/style/_query-builder-property-search-panel.scss

```diff
@@ -140,6 +140,15 @@
     }
   }
 
+  &__search__mode {
```

Comment:
hm, if we modularize, I don't think we would expect this many changes in this file

Reply:
Modularize search count, cog icon, and search icon as well? 

<img width="101" alt="image" src="https://user-images.githubusercontent.com/41248956/191663661-6b1180c9-0b74-4002-b63b-b3c2da868aca.png">

Reply:
nope